index/index.go

```diff
@@ -852,9 +866,13 @@ func (r *Reader) LabelIndices() ([][]string, error) {
 	return res, nil
 }
 
-// Series the series with the given ID and writes its labels and chunks into lbls and chks.
+// Reads the series with the given ID and writes its labels and chunks into lbls and chks.
```

Comment:
In go the comments need to start with the function name. `Series reads the.....`